mutation/canonical_mutation.cc

```diff
@@ -67,6 +67,34 @@ mutation canonical_mutation::to_mutation(schema_ptr s) const {
     return m;
 }
 
+future<mutation> canonical_mutation::to_mutation_gently(schema_ptr s) const {
+    auto in = ser::as_input_stream(_data);
```

Comment:
Ditto, we may want to make this an external utility.

Reply:
okay, I'll try refactoring it out to a separate file